recipes/app.js

```diff
@@ -1,6 +1,7 @@
 var express = require('express');
```

Comment:
Whichever variable doesn't change reference should be `const`. If it does, then use `let`. There is seldom an usecase for `var`